packages/world/index.yml

```diff
@@ -0,0 +1,2 @@
+label: store
```

Comment:
using world to align `schema-types` package to it. Noticed that this should probably be `world`